src/page-module/files/__name@dasherize__/__name@dasherize__.component.spec.ts

```diff
@@ -33,7 +34,7 @@ describe('<%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + nam
         <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageComponent
       ],
       routes: [],
-      providers: []
+      providers: [<% if (facade) {%><%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageFacade<% } %>]
```

Comment:
@Goodmain same as above, this `classify` call can be shortened.

Reply:
@dmitryusenko Fixed